src/components/List/List.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './List.module.scss';
+import ListItem from './ListItem';
+
+const List = ({ items }) => (
+	<ul className={styles.itemsWrapper}>
+		{items.map((item, index) => (
```

Comment:
Never use the `index` as the key. Try to use an unique value of the object you're rendering. E.g: `item.id`, `item.name` or a combination that would produce an unique value. E.g: `item.category + item.name`